api/src/ginger_calls.rs

```diff
@@ -1,85 +1,111 @@
 use algebra::{fields::{
-    mnt4753::{Fr, Fq as ScalarFieldElement}, PrimeField
+    tweedle::{Fq as ScalarFieldElement, Fr}, PrimeField
 }, curves::{
-    mnt4753::MNT4 as PairingCurve,
-    mnt6753::{
-        G1Projective as GroupProjective, G1Affine as GroupAffine
+    tweedle::{
+        dum::{
+            Projective as Projective,
+            Affine as Affine
+        },
+        dee::Affine as PCAffine
     },
-}, FromBytes, ToBytes, BigInteger768, ProjectiveCurve, AffineCurve, ToConstraintField, UniformRand, ToBits};
+}, FromBytes, FromBytesChecked, validity::SemanticallyValid,
+   ToBytes, BigInteger256, ProjectiveCurve, AffineCurve, ToConstraintField, UniformRand, ToBits};
 use primitives::{crh::{
-    poseidon::{
-        MNT4PoseidonHash,
-        batched_crh::MNT4BatchPoseidonHash as BatchFieldHash,
-    },
+    poseidon::parameters::tweedle::{TweedleFrPoseidonHash as PoseidonHash, TweedleFrBatchPoseidonHash as BatchFieldHash},
     FieldBasedHash,
     bowe_hopwood::{
         BoweHopwoodPedersenCRH, BoweHopwoodPedersenParameters
     },
 }, merkle_tree::field_based_mht::{
-    smt::{BigMerkleTree, LazyBigMerkleTree, Coord, OperationLeaf},
+    // smt::{BigMerkleTree, LazyBigMerkleTree, Coord, OperationLeaf},
```

Comment:
@DanieleDiBenedetto so now we have commented BigMerkleTree and no dependency on `smt` feature of ginger lib.
Am I right, that now there is no dependency on rocks DB, so no problem with libstd dependencies on Linux and Windows?